### DISCUSSION TITLE
Support all aspects of LogOutputViewChannel (as in VSCode)

### DISCUSSION THREAD

**Discussion by Hemavathy12** [eclipse-theia/theia#13305] [Q&A]
VSCode writes the logs to a separate file per extension and offers a command and tool in the output view to directly open the extension's log file. It would be better if we add support for all aspects of LogOutputViewChannel (as in VSCode) (i.e. developer logger service, logging to file, extension of output view UI).

I am developing an application based out of Theia. Need the above mentioned logging capabilities. Is there any alternative way to include the above mentioned logging features to my application?

**Comment by msujew** [comment]
@Hemavathy12 I've opened an issue from this, see https://github.com/eclipse-theia/theia/issues/13307. I think this is better suited as a feature request.
